include/nbl/video/IDeviceMemoryBacked.h

```diff
@@ -42,8 +44,21 @@ class IDeviceMemoryBacked : public IBackendObject
             {
                 return queueFamilyIndexCount>1u;
             }
+
+            //! Note that this will return false for the special foreign and external families
+            inline bool canBeUsedByQueueFamily(const uint32_t family) const
+            {
+                if (!isConcurrentSharing())
+                    return true;
+                assert(queueFamilyIndices);
+                for (uint8_t f=0; f<queueFamilyIndexCount; f++)
+                if (queueFamilyIndices[f]==family)
+                    return true;
+                return false;
+            }
         };
```

Comment:
@Devsh-Graphics-Programming/seniors @Devsh-Graphics-Programming/mids now one can query the concurrent families the object was created with